nzediegwu1/EventManager
# #157219929 Refactor client source code

#### What does this PR do?
Refactor client source code based on feedback and recommendations

#### Description of Task to be completed?
1. Created services.js to handle all http transactions.
2. Move reusables.js into service.js and deleted reusables.js
3. Make client/src/resources/images static in app.js for easy reference in jsx image srcs
4. Refactor addCenterComponent.js to eliminate images imports
5. Refactor addCenterComponent.js to handle http requests in services.js
6. Refactor addEventComponent.js to eliminate images imports
7. Refactor addEventComponent.js to handle http requests in services.js
8. Create Icon component to render icons images
9. Refactor centerDetails component to use Icon component and handle http requests in services.js
10. Refactor centerList component.js to handle http requests in services.js
10. Refactor eventList component.js to handle http requests in services.js
11. Refactor formGroup component to allow use of image name directly in src rather than imported image
12. Refactor manageDetailsHeader component to use Icon component and handle http requests in services.js
13. Refactor manageEvent component to use Icon component and handle http requests in services.js
14. Refactor manageFacilities component to use Icon component and handle http requests in services.js
15. Refactor navbar component to use Icon component
16. Refactor profile component to handle http requests in services.js
17. Refactor sidebar and recoverPassword components to use Icon component
18. Refactor signin and signup components to use Icon component and handle http requests in services.js
19. Refactor userList component to handle http requests in services.js

#### How should this be manually tested?
On the terminal, run command:
1. `npm run start`
2. `npm run client`
3. On the browser, login to the app and test features affected to ensure that they still work as before

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories? (If applicable)
[#157219929](https://www.pivotaltracker.com/story/show/157219929)

#### Screenshots (If applicable)
